nbs/tutorials/methods.ipynb

```diff
@@ -11,10 +11,43 @@
    "cell_type": "markdown",
```

Comment:
<p>What is this <code>num_cfs</code>? I don't think VanillaCF has this argument.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/jax-relax/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
corrected the arguments